### ISSUE TITLE
[BUG] WebKit on Mac crashes when RTCPeerConnection is used.

### ISSUE THREAD

**Issue by mxschmitt** [microsoft/playwright#6009]
**Context:**
- Playwright Version: 1.10
- Operating System: Mac 11.2.2 and 10.15.7
- Node.js version: v14.8.0
- Browser: WebKit
- Original issue: https://github.com/playwright-community/jest-playwright/issues/659

**To Reproduce**

Execute this script:

```js
// @ts-check
const playwright = require('playwright');

(async () => {
  const browser = await playwright.webkit.launch();
  const context = await browser.newContext();
  const page = await context.newPage();
  await page.goto("https://www.whatismybrowser.com/")
  await page.screenshot({ path: `example.png` });
  await browser.close();
})();
```

or from the original bug report:

1. `git clone https://github.com/jsjoeio/jest-playwright-repro.git`
2. `yarn`
3. `yarn test`

**Describe the bug**

Actual: WebKit page crashes with `Navigation failed because page crashed`
Expected: Test passes

It works for Chromium and Firefox.

~~It seems like its because `https://whatismybrowser.com/` is a redirect -> 301 redirect.~~

**Comment by mxschmitt** [comment]
Seems like related to their JavaScript:

```
it.only("foo", async ({ page, server }) => {
  await page.route("https://cdn.whatismybrowser.com/prod-website/static/main/js/site.min.js**", route => {
    return route.abort()
  })
  await page.goto('https://www.whatismybrowser.com/')
  expect(page.url()).toBe('https://www.whatismybrowser.com/')
});
```

this passes.

**Comment by jsjoeio** [comment]
When will this fix land?

**Comment by yury-s** [comment]
It's fixed in @next and will be released in v1.11.0
